test/e2e/specs/editor/various/adding-patterns.spec.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'adding patterns', () => {
+	test.beforeEach( async ( { admin } ) => {
+		await admin.createNewPost();
+	} );
+
+	test( 'should insert a block pattern', async ( { page, editor } ) => {
+		await page.click(
+			'role=region[name="Editor top bar"i] >> role=button[name="Toggle block inserter"i]'
+		);
+
+		await page.fill(
+			'role=region[name="Block Library"i] >> role=searchbox[name="Search for blocks and patterns"i]',
+			'Social links with a shared background color'
+		);
+
+		await page.click(
+			'role=option[name="Social links with a shared background color"i]'
+		);
+		await expect.poll( editor.getBlocks ).toMatchObject( [
+			{
+				name: 'core/social-links',
+				attributes: { className: 'has-icon-color' },
```

Comment:
Nit: I don't think we need this. I think we only want to check if it's a `social-links` block.

```suggestion
```